scalableminds/webknossos
# Add fill tool

This PR adds a flood fill tool to the brush.

TODOS:
- [x] use a better shortcut for the flood fill tool. -> replace with automatic brush
- [x] test this with a fallback layer -> works
- [x] integrate with undo redo

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open chrome (firefox is not supported)
- Open a volume tracing and draw a circle in two strokes (not one as this would fill the whole inner area)
- Use "win/cmd" + left mouse inside the circle
- The area should be filled with the active cell id (should be pretty fast)

### Issues:
- fixes #4713

------
